enterprise/views.py

```diff
@@ -75,15 +77,21 @@ def verify_edx_resources():
     """
     Ensure that all necessary resources to render the view are present.
     """
-    required_methods = (
-        configuration_helpers,
-        get_enterprise_customer_for_request,
-        get_real_social_auth_object,
-        quarantine_session,
-        lift_quarantine
-    )
-    if any(method is None for method in required_methods):
-        raise NotConnectedToEdX(_('Methods in the Open edX platform necessary for this view are not available.'))
+    required_methods = {
+        'configuration_helpers': configuration_helpers,
+        'get_enterprise_customer_for_request': get_enterprise_customer_for_request,
+        'get_real_social_auth_object': get_real_social_auth_object,
+        'quarantine_session': quarantine_session,
+        'lift_quarantine': lift_quarantine,
+        'get_partial_pipeline': get_partial_pipeline,
+    }
```

Comment:
@bradenmacdonald I actually agree with the above solutions, but I've been beating this exception bush quite a bit in this PR and I think I want to just dedicate a new, smaller PR with a cleaner approach to this; for now it'll do, if that's alright -- but definitely want to change it.